app/styles/main.scss

```diff
@@ -163,72 +160,70 @@ line-height:0.8em;
     visibility: hidden;
 }
 
-
-
-
-
 // =============== ball animations =============
 
-@keyframes firstarch {
-    from {transform: rotate(-25deg); transform-origin: bottom right;    }
-    to {transform: rotate(75deg);  transform-origin: bottom right; }
-    // 100% {transform: rotate(70deg)translateY(-200px); }
-      
+@keyframes horizontal-motion {
+    from {  }
+    to { transform: translateX(800px); }
 }
 
- @keyframes secondarch {
-    from { transform: rotate(75deg) translateX(0px) translateY(0px);  transform-origin: bottom right;    }
-     to {  transform: rotate(160deg) translateX(-460px) translateY(-420px); transform-origin: top right;  }
- }
+@keyframes spin {
+    from {  }
+    to { transform: rotate(calc(360deg * (800 / (50 * 3.14159)))); } //
+}
 
- @keyframes thirdarch {
-    from {  transform: rotate(160deg) translateX(-460px) translateY(-420px); transform-origin: top right;  }
-    to { transform: rotate(200deg) translateX(-900px) translateY(70px); transform-origin: top left; }
- }
-
- @keyframes fourtharch {
-    from {transform: rotate(200deg) translateX(-900px) translateY(70px); transform-origin: top left;}
-    to{transform: rotate(230deg) translateX(-675px) translateY(1100px); transform-origin: bottom left;}
- }
+// Heights and timings have been calculated off-screen to match real-world
+// physics for a reasonably bouncy (90%) ball.
 
+@keyframes firstbounce {
+    from {  }
+    to { margin-top: 0px }
+}
 
+@keyframes secondbounce {
+    from {  }
+    to { margin-top: 25px }
```

Comment:
FYI, animating `transform` is cheaper than animating `margin`because of the way the repainting engine works

Reply:
The problem I discovered is, because I'm already using `transform` in `spin`, having two different keyframe animations that both use `transform` breaks both of them.